webgpu.yml

```diff
@@ -22,44 +22,53 @@ constants:
   - name: array_layer_count_undefined
     value: uint32_max
     doc: |
-      TODO
+      Indicates no array layer count is specified. For more info,
+      see @ref SentinelValues and the places that use this sentinel value.
   - name: copy_stride_undefined
     value: uint32_max
     doc: |
-      TODO
+      Indicates no copy stride is specified. For more info,
+      see @ref SentinelValues and the places that use this sentinel value.
   - name: depth_clear_value_undefined
     value: nan
     doc: |
-      Value to be assigned to member depthClearValue of @ref WGPURenderPassDepthStencilAttachment
-      to mean that it is not defined.
+      Indicates no depth clear value is specified. For more info,
+      see @ref SentinelValues and the places that use this sentinel value.
   - name: depth_slice_undefined
     value: uint32_max
     doc: |
-      TODO
+      Indicates no depth slice is specified. For more info,
+      see @ref SentinelValues and the places that use this sentinel value.
   - name: limit_u32_undefined
     value: uint32_max
     doc: |
-      TODO
+      Indicates no limit value is specified. For more info,
+      see @ref SentinelValues and the places that use this sentinel value.
   - name: limit_u64_undefined
     value: uint64_max
     doc: |
-      TODO
+      Indicates no limit value is specified. For more info,
+      see @ref SentinelValues and the places that use this sentinel value.
   - name: mip_level_count_undefined
     value: uint32_max
     doc: |
-      TODO
+      Indicates no mip level count is specified. For more info,
+      see @ref SentinelValues and the places that use this sentinel value.
   - name: query_set_index_undefined
     value: uint32_max
     doc: |
-      TODO
+      Indicates no query set index is specified. For more info,
+      see @ref SentinelValues and the places that use this sentinel value.
   - name: whole_map_size
     value: usize_max
     doc: |
-      TODO
+      Indicates a request for the maximum size. For more info,
```

Comment:
```suggestion
      Indicates a request for the whole buffer to be mapped. For more info,
```

Reply:
How about `Indicates a size extending to the end of the buffer.`?